src/d3a/models/strategy/storage.py

```diff
@@ -589,9 +589,10 @@ def event_offer(self, *, market_id, offer):
             market = self.area.get_future_market_from_id(market_id)
             # sometimes the offer event arrives earlier than the market_cycle event,
             # so the default values have to be written here too:
+            # TODO: Evaluate for removal
```

Comment:
Indeed!

Reply:
I just recalled that I had added it there! I'll actually deal with it in this PR